.github/workflows/build-nightly.yml

```diff
@@ -41,9 +41,14 @@ jobs:
           labels: |
             org.opencontainers.image.description=Nightly build image used for testing purposes
 
+      - name: Collect variables for current date
+        id: date_vars
+        run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
```

Comment:
Wouldn't it be better to do this the same way as in build-image? So:

```
        run: echo "MASTODON_VERSION_SUFFIX=+nightly-$(date +'%Y%m%d')" >> $GITHUB_ENV
```

Though it might be nicer to use `$GITHUB_OUTPUT` instead of `$GITHUB_ENV` consistently between the two workflows.

Reply:
I initially used `$GITHUB_OUTPUT` everywhere as its cleaner, but for the `edge` builds the step is conditional, and I am unsure how it will behave if the variable is not set, hence the change to `GITHUB_ENV`.

I think we could unify both to use `GITHUB_OUTPUT` and set the whole string there, but this needs to be tested and I was not sure of the best way to do it.

Reply:
Wouldn't both just become:

```
build-args: MASTODON_VERSION_SUFFIX=${{ steps.determine_version.outputs.version_suffix }}
```

and 

```
     - name: Prepare the version number for the docker image
        id: determine_version
        run: echo "version_suffix=+nightly-$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
// OR
        run: echo "version_suffix=+edge-$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```

As the step not being run would result in null output, hence `''` for `MASTODON_VERSION_SUFFIX`

Reply:
Pushed a new version with a similar idea, let's see if it runs fine